localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears whispers of the Old Gods.;;;;;;;;;;;;;x
+whispers_tooltip_tip;Quality of the book depends on amount of your �YTrueReligion.GetPietyName�!.;;;;;;;;;;;;;x
+EVTDESC_WCCSO.10200;The whispers of your masters have been getting stronger recently. They are pouring their eldritch knowledge into your mind but it's simply too much for your mind to take and you're unable to make sense of any of it. You need to focus on one portion of them and write them down, but first you need a writing material.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCSO.10200;The whispers of your masters have been getting stronger recently. They are pouring their eldritch knowledge into your mind, but it's simply too much for your mind to take and you're unable to make sense of any of it. You need to focus on one portion of the whispers and write them down, but first you need writing materials.;;;;;;;;;;;;;x
```

Reply:
Is "portion" really the right word here?

Reply:
Honestly, you are right, but not sure what to replace it with. One part of the whispers? One particular whisper?

Reply:
Piece or Part maybe? Aspect? Fragment?

Reply:
One fragment might be good. I don't think aspect could work though. Piece and part could also work, although I think part is better than piece.